autoPyTorch/pipeline/components/training/trainer/cutout_utils.py

```diff
@@ -0,0 +1,144 @@
+from typing import Any, Callable, Dict, Optional
+
+from ConfigSpace.conditions import EqualsCondition
+from ConfigSpace.configuration_space import ConfigurationSpace
+from ConfigSpace.hyperparameters import (
+    CategoricalHyperparameter,
+    Constant,
+    UniformFloatHyperparameter,
+)
+
+import numpy as np
+
+from sklearn.utils import check_random_state
+
+from autoPyTorch.constants import CLASSIFICATION_TASKS, STRING_TO_TASK_TYPES
+from autoPyTorch.pipeline.components.training.trainer.utils import Lookahead
+from autoPyTorch.utils.common import HyperparameterSearchSpace, add_hyperparameter, get_hyperparameter
+
+
+class CutOut:
```

Comment:
I do not feel the necessity of the separation of `CutOut` and `MixUp` since 95% of the code is same.
Can't we just merge them by using `**kwargs`?